transformer/src/main/scala/com/snowplowanalytics/snowflake/transformer/TransformerJob.scala

```diff
@@ -71,6 +91,15 @@ object TransformerJob {
     // DataFrame is used only for S3OutputFormat
     snowflake.toDF.write.mode(SaveMode.Append).text(jobConfig.output)
 
+    // Write out errorEvents if an errorOutput location is defined
+    jobConfig.errorOutput match {
+      case Some(path) if !errorEvents.isEmpty() =>
+        errorEvents.toDF.write.mode(SaveMode.Append).text(path)
+        log.info(s"Writing error events to $path.")
+      case Some(_) => log.info(s"No error events for ${jobConfig.output}.")
```

Comment:
Maybe I'm confusing myself here, but in this case you do have error events, but no path defined to output it to, right? So, in that case the logs for this line and the following should be switched around.

Reply:
Yeah, I think I've confused myself. Still, shouldn't you rather cover this case as the `else` counterpart of the `if`-statement in line 96? Would make it clearer. Unless I'm still confused. 😛

Reply:
I agree this is confusing, will try to simplify this, thanks (: